<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemWidget</class>
 <widget class="QWidget" name="ItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ICEGUI_NS::EditedGroupBox" name="maskGB">
     <property name="title">
      <string>Эталон</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="yMaskL">
              <property name="text">
               <string>Отношение y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yMaskDSB">
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="singleStep">
               <double>0.001000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="maskCalcPB">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Рассчитать</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="yKMaxMaskL">
              <property name="text">
               <string>Кв:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yKMaxMaskDSB">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.001000000000000</double>
              </property>
              <property name="maximum">
               <double>99.998999999999995</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>1.100000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="yKMaskL">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>k:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yKMaskDSB">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.001000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="yKMaskChB">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="yKMinMaskL">
              <property name="text">
               <string>Кн:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="yKMinMaskDSB">
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.001000000000000</double>
              </property>
              <property name="maximum">
               <double>99.998999999999995</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.900000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="maskAutoKPB">
              <property name="text">
               <string>Авто</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="maskClearKPB">
              <property name="text">
               <string>Сброс</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QPushButton" name="maskCancelPB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>28</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="ICEGUI_NS::MaskViewWidget" name="maskViewWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ICEGUI_NS::MaskViewWidget</class>
   <extends>QWidget</extends>
   <header location="global">icegui/maskviewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ICEGUI_NS::EditedGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">icegui/editedgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../iceavglumproc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>maskCalcPB</sender>
   <signal>clicked()</signal>
   <receiver>ItemWidget</receiver>
   <slot>maskCalcPB_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maskCancelPB</sender>
   <signal>clicked()</signal>
   <receiver>ItemWidget</receiver>
   <slot>maskCancelPB_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maskAutoKPB</sender>
   <signal>clicked()</signal>
   <receiver>ItemWidget</receiver>
   <slot>maskAutoKPB_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maskClearKPB</sender>
   <signal>clicked()</signal>
   <receiver>ItemWidget</receiver>
   <slot>maskClearKPB_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yKMaskChB</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ItemWidget</receiver>
   <slot>yKMaskChB_stateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yKMaskDSB</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ItemWidget</receiver>
   <slot>yKMaskDSB_valueChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maskGB</sender>
   <signal>toggled(bool)</signal>
   <receiver>ItemWidget</receiver>
   <slot>maskGB_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>maskCalcPB_clicked()</slot>
  <slot>maskCancelPB_clicked()</slot>
  <slot>maskAutoKPB_clicked()</slot>
  <slot>maskClearKPB_clicked()</slot>
  <slot>yKMaskDSB_valueChanged(double)</slot>
  <slot>yKMaskChB_stateChanged(int)</slot>
  <slot>maskGB_toggled(bool)</slot>
 </slots>
</ui>
